SolidStateGroup/bullet-train-api
# fix: some emails cannot reset their passwords

Thanks for submitting a PR! Please check the boxes below:


## Changes

- In the frontend I have updated the regex used for email validation to the one found on https://www.regextester.com/115911

- resolves #2004 

## How did you test this code?

1. run `npm dev`
2. Navigate to SignUp
3. Click "Forgot Password"
4. Enter the following email address `user@company.fashion`.

You should expect to see the "Send" button in an active state, as it is a valid email address.

Previously the "Send" button would remain inactive preventing us from resetting our accounts.
